pouchdb-core/pouchdb-core-tests.ts

```diff
@@ -40,9 +40,9 @@ namespace PouchDBCoreTests {
 
     function testBulkDocs() {
         const db = new PouchDB<MyModel>();
-        type MyModel = { property: 'someProperty '};
-        let model: PouchDB.Core.Document<MyModel>;
```

Comment:
I was intending to be explicit rather than relying on inference.  Your change works right now, but as types evolve I'm not sure it'll be as useful of a test.